src/mixins/filing-template-mixin.ts

```diff
@@ -413,9 +440,15 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         validationItemDetails: []
       },
       resolutionConfirmed:
-        draftFiling.dissolution?.specialResolution?.resolutionConfirmed ||
+        draftFiling?.specialResolution?.resolutionConfirmed ||
         draftFiling.dissolution?.resolution?.resolutionConfirmed || false
     }
+    if (draftFiling.specialResolution) {
+      createResolution.resolutionDate = draftFiling.specialResolution?.resolutionDate
+      createResolution.resolutionText = draftFiling.specialResolution?.resolution
+      createResolution.signingPerson = draftFiling.specialResolution?.signatory
+      createResolution.signingDate = draftFiling.specialResolution?.signingDate
```

Comment:
You don't need conditional chaining here since you verified that the object exists on line 446.

I wonder if restructuring would simplify this code? But I'm OK with this too.

Reply:
will take a look.

Reply:
have updated to remove conditional chaining but as mentioned above, will be taking a second look at code around here in subsequent PR.